pages/windows/mkdir.md

```diff
@@ -7,6 +7,6 @@
 
 `mkdir {{directory_name}}`
 
-- Recursively create a nested directory tree:
+- Create a nested directory tree recursively:
 
-`mkdir {{path/to/sub_directory_name}}`
+`mkdir --parents {{path/to/sub_directory_name}}`
```

Comment:
I tested `mkdir` in WIndows and this example creates 2 paths, a directory named `--parents`, and `path/to/dir`.
`mkdir` doesn't accept argument, and `mkdir --help` creates a `--help` directory. The directory tree is created without need of an extra argument. so:
```suggestion
`mkdir {{path/to/sub_directory}}`
```

Reply:
Ahh shoot… I'm an idiot.
I thought I saw this in the common directory. :facepalm: